### PR TITLE
ACMS-1609: Adding is_syncing for hook_install, hook_modules_installed…

### PR DESCRIPTION
**Pull request by apathak18**
… and its uninstall.

**Motivation**

Fixes ACMS-1609

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by deepakmishra2 — APPROVED** [review]
LGTM

File: modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php [hunk 1]
```diff
@@ -64,134 +64,136 @@ public static function getSubscribedEvents() {
...
-          case 'core.entity_view_display.node.page.default':
-            _acquia_cms_common_update_page_configurations('core.entity_view_display.node.page.default', [
-              'content.field_layout_canvas' => [
-                'type' => 'cohesion_entity_reference_revisions_entity_view',
-                'weight' => 2,
-                'label' => 'hidden',
-                'settings' => [
-                  'view_mode' => 'default',
-                  'link'  => FALSE,
+    if (!\Drupal::isConfigSyncing()) {
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php:67]: @apathak18 Can you please confirm, if this is working as expected when you run `drush cim` ?

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

**Review by chandan-singh7929 — APPROVED** [review]
This looks good to me, though there is one test failure, need to check that.

File: modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php (hunk 1 above)
Comment (apathak18) [reply modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php]: Reverted this one

**Review by rajeshreeputra — APPROVED** [review]
Overall changes looks good.

**Pull request merged**